gatsby/static/css/matrix-org.webflow.css

```diff
@@ -1358,9 +1354,6 @@ a {
   .mxnavbar__navlink.mxnavbar__navlink--primary {
     display: none;
   }
-  .mxnavbar__navlink.mxnavbar__navlink--try {
-    display: block;
-  }
```

Comment:
We also need to remove the [`display: none` rule for `.mxnavbar__navlink.mxnavbar__navlink--primary`](https://github.com/matrix-org/matrix.org/blob/8a2a43545a19e841542635d9c97ecae18e8f1cca/gatsby/static/css/matrix-org.webflow.css#L1358-L1360) so it shows up in the hamburger mobile menu.

It's currently missing with that rule in place:

<img width="570" alt="Screen Shot 2021-08-16 at 9 49 15 PM" src="https://user-images.githubusercontent.com/558581/129656245-a31d3a2c-d5de-4bfb-af62-9923a0668e5d.png">

Reply:
Good catch, fixed.